SimplenoteScreenshots/SnapshotHelper.swift

```diff
@@ -219,7 +227,7 @@ open class Snapshot: NSObject {
         #if os(OSX)
             let homeDir = URL(fileURLWithPath: NSHomeDirectory())
             return homeDir.appendingPathComponent(cachePath)
-        #elseif arch(i386) || arch(x86_64)
+        #elseif arch(i386) || arch(x86_64) || arch(arm64)
```

Comment:
This line made me go into another 🕳️ , and learn some new things! Nice!

Reply:
While I was reading about this, it saw such examples. Looks like this might do the similar trick?

```suggestion
        #elseif targetEnvironment(simulator)
```

I tested this locally and it worked. I might miss some context, though 🤔

Reply:
This code is automatically generated by Fastlane, so I'm not sure about the design decisions the authors took. Also, we should not touch it because the screenshots automation expects it work in a certain way.

Is your to use `targetEnvironment(simulator)` because it's platform agnostic? I think it's a great idea. You should open a pull request for that in Fastlane itself. The source for this file is [here](https://github.com/fastlane/fastlane/blob/master/snapshot/lib/assets/SnapshotHelper.swift).

👍

Reply:
> You should open a pull request for that in Fastlane itself.

This sounds exciting! Yes, it is to make it platform-agnostic